test-cypress.sh

```diff
@@ -68,7 +68,7 @@ if [ -n "${headless-}" ] && [ -z "${pkg-}" ]; then
   yarn run test-cypress-dev-console-headless
   yarn run test-cypress-olm-headless
   yarn run test-cypress-helm-headless
-  yarn run test-cypress-knative-headless
+# yarn run test-cypress-knative-headless
```

Comment:
Let's make sure we have an issue tracking reenabling these tests, cc @invincibleJai